src/utils/build.js

```diff
@@ -106,6 +108,39 @@ const listFiles = dir => {
   });
 };
 
+// returns paths as to be chainable
+const verifyNodeFeatures = paths => {
+  const opts = {
+    plugins: ['node'],
+    // set version as high as possible, let's parse it all
+    // see: https://eslint.org/docs/user-guide/configuring#specifying-parser-options
+    parserOptions: { ecmaVersion: 9 },
+    useEslintrc: false, // literally only check this rule
+    rules: {
+      'node/no-unsupported-features': [
+        'error',
+        { version: require('../constants').LAMBDA_VERSION }
+      ]
+    }
+  };
+
+  // only lint js files; node_modules ignored by default
+  const jsPaths = paths.filter(p => p.endsWith('.js'));
+  const cli = new eslint.CLIEngine(opts);
+  const errors = eslint.CLIEngine.getErrorResults(
+    cli.executeOnFiles(jsPaths).results
+  );
+
+  if (errors.length) {
+    if (!process.env.NODE_ENV === 'test') {
```

Comment:
I think you meant to write `process.env.NODE_ENV !== 'test'`. Otherwise, the errors will never be printed.

Reply:
ah, great catch! this isn't honestly that important, I just got annoyed with eslint errors that are supposed to fail showing up and muddying up the test.